test/abilities/shields-down.test.ts

```diff
@@ -54,4 +56,130 @@ describe("Abilities - SHIELDS DOWN", () => {
 
     expect(minior.formIndex).toBe(meteorForm);
   });
+
+  test("should ignore non-volatile status moves", async () => {
+    game.override.enemyMoveset([MoveId.SPORE]);
+
+    await game.classicMode.startBattle([Species.MINIOR]);
+    game.move.use(MoveId.SPLASH);
+    await game.phaseInterceptor.to(TurnEndPhase);
+
+    expect(game.field.getPlayerPokemon()!.status).toBe(undefined);
+  });
+
+  test("should still ignore non-volatile status moves used by a pokemon with mold breaker", async () => {
+    game.override.enemyAbility(Abilities.MOLD_BREAKER);
+    game.override.enemyMoveset([MoveId.SPORE]);
+
+    await game.classicMode.startBattle([Species.MINIOR]);
+
+    game.move.use(MoveId.SPLASH);
+    await game.move.forceEnemyMove(MoveId.SPORE);
+    await game.phaseInterceptor.to(TurnEndPhase);
+
+    expect(game.field.getPlayerPokemon()!.status).toBe(undefined);
+  });
+
+  test("should ignore non-volatile secondary status effects", async () => {
+    game.override.enemyMoveset([MoveId.NUZZLE]);
+
+    await game.classicMode.startBattle([Species.MINIOR]);
+
+    game.move.use(MoveId.SPLASH);
+    await game.phaseInterceptor.to(TurnEndPhase);
+
+    expect(game.field.getPlayerPokemon()!.status).toBe(undefined);
+  });
+
+  test("should ignore status moves even through mold breaker", async () => {
+    game.override.enemyMoveset([MoveId.SPORE]);
+    game.override.enemyAbility(Abilities.MOLD_BREAKER);
+
+    await game.classicMode.startBattle([Species.MINIOR]);
+
+    game.move.use(MoveId.SPLASH);
+
+    await game.phaseInterceptor.to(TurnEndPhase);
+
+    expect(game.field.getPlayerPokemon()!.status).toBe(undefined);
+  });
+
+  // toxic spikes currently does not poison flying types when gravity is in effect
+  test.todo("should become poisoned by toxic spikes when grounded", async () => {
+    game.override.enemyMoveset([MoveId.GRAVITY, MoveId.TOXIC_SPIKES, MoveId.SPLASH]);
+    game.override.moveset([MoveId.GRAVITY, MoveId.SPLASH]);
+
+    await game.classicMode.startBattle([Species.MAGIKARP, Species.MINIOR]);
+
+    // turn 1
+    game.move.use(MoveId.GRAVITY);
+    await game.move.forceEnemyMove(MoveId.TOXIC_SPIKES);
```

Comment:
using `enemyMoveset` and then using `forceEnemyMove` is redundant, unless the forced move is not in the moveset
```suggestion
    await game.move.selectEnemyMove(MoveId.TOXIC_SPIKES);
```